scipy/optimize/_minimize.py

```diff
@@ -40,6 +40,7 @@
                     'l-bfgs-b', 'tnc', 'cobyla', 'slsqp', 'trust-constr',
                     'dogleg', 'trust-ncg', 'trust-exact', 'trust-krylov']
 
+MINIMIZE_SCALAR_METHODS = ['Brent', 'Bounded', 'Golden']
```

Comment:
```suggestion
MINIMIZE_SCALAR_METHODS = ['brent', 'bounded', 'golden']
```
Not at all important, but this would be more consistent.